### PR TITLE
Update natural-language config to Mistral

### PR DESCRIPTION
**Pull request by lipikaramaswamy**
Update `config_templates/gretel/synthetics/natural-language.yml`
* Use updated pretrained model `mistralai/Mistral-7B-Instruct-v0.2` 
* Lower learning rate to 1e-4  / 0.0001
  * This value was chosen using Gretel Tuner on 4 public example notebooks, maximizing for Text SQS. It is also consistent with observations from the community that fine tuning mistral 7b is more effective with a lower learning rate.

### CODE REVIEW

**Review by mvansegbroeck — APPROVED** [review]
Tested out his model on this [notebook](https://github.com/gretelai/gretel-blueprints/blob/main/docs/notebooks/demo/gretel-demo-create-synthetic-dialogs.ipynb) and results are all looking great
